examples/arguments/arguments_example.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"fmt"
-	"github.com/docopt/docopt-go"
+	"github.com/docopt/docopt.go"
```

Comment:
As we say in the [README](https://github.com/docopt/docopt.go#installation), it is preferable to import the alias `docopt-go` since having a dot `.` in the package name doesn't play nicely with all Go tools.